src/main/java/org/mskcc/cbio/oncokb/service/MailService.java

```diff
@@ -94,7 +94,7 @@ public void sendEmail(String to, String from, String cc, String subject, String
         MimeMessage mimeMessage = javaMailSender.createMimeMessage();
         MimeMessageHelper message = new MimeMessageHelper(mimeMessage, isMultipart, StandardCharsets.UTF_8.name());
         message.setTo(to);
-        if (cc != null) {
+        if (StringUtils.isNotEmpty(cc)) {
```

Comment:
Similar to this change, should we update to use `isNotEmpty(sendBy)` on this [line](https://github.com/oncokb/oncokb-public/blob/3695c00c689ed983b2b1404a15669debd24c6631/src/main/java/org/mskcc/cbio/oncokb/service/MailService.java#L193) too?

Reply:
@calvinlu3 good call. Made the change.